Ginger/GingerCoreCommon/GeneralLib/BetaFeatures.cs

```diff
@@ -63,6 +63,7 @@ public class BetaFeatures : INotifyPropertyChanged
 
         // ALM
         public bool Rally { get { return GetFeature(nameof(Rally)).Selected; } set { UpdateFeature(nameof(Rally), value); } }
+        public bool RestAPI { get { return GetFeature(nameof(RestAPI)).Selected; } set { UpdateFeature(nameof(RestAPI), value); } }
 
```

Comment:
Beta flag can be removed now because QC Rest feature is 95% ready according to you